src/components/Loading.tsx

```diff
@@ -137,7 +137,8 @@ const Loading = ({ children, className, dataTestId }: LoadingProps) => {
             __html: Marked.parse(
               CONNECTION_ERROR_TEXT[error.error] +
                 (error.context
-                  ? '\n\nThe error details are: ' + error.context
+                  ? '\n\nThe error details are: ' +
+                    JSON.stringify(error.context)
```

Comment:
Unrelated, I hit this error and it reads `The error details are: [object: Object]` figure it was better that it's somewhat readable.